model/export/SingleDeliveryResultsExporter.php

```diff
@@ -162,10 +162,6 @@ public function getColumnsToExport()
                 );
             }
 
-            // Needed by the filter to filter by start and end date
```

Comment:
Why you did remove this lines?

Reply:
Because they were the reason why the bug happened - these fields added to the export regardless of the export setup.